apisix/plugins/wolf-rbac.lua

```diff
@@ -284,7 +283,8 @@ function _M.rewrite(conf, ctx)
 
     local consumer_conf = consumer.plugin(plugin_name)
     if not consumer_conf then
-        return 401, fail_response("Missing related consumer")
+        core.log.warn("Missing related consumer")
+        return 401, fail_response("Invalid user authorization")
```

Comment:
the error here is nothing about the Authorization header

Reply:
I'll reset this modification